multisig/simulations/arbitrumone/vip-011/index.ts

```diff
@@ -0,0 +1,11 @@
+import { forking, pretendExecutingVip } from "src/vip-framework";
+
+import vip011 from "../../../proposals/arbitrumone/vip-011";
+
+forking(245111060, async () => {
+  describe("Post-VIP behavior", async () => {
+    it("execute vip", async () => {
+      await pretendExecutingVip(await vip011());
+    });
```

Comment:
I think, we should add test to verify that role has been renounced successfully.